### ISSUE TITLE
WCAG 2.2: Contact a department or service team ‖ Consistent Help

### ISSUE THREAD

**Issue by querkmachine** [alphagov/govuk-design-system#2882]
## What

Update our guidance on how to contact departments and service teams to emphasise that if contact information is present on multiple pages, the types of contact information should be listed in a consistent order across all of those pages. 

This is largely a usability suggestion and may not be necessary for WCAG compliance. 

### Resources

- [Contact a department or service team pattern guidance](https://design-system.service.gov.uk/patterns/contact-a-department-or-service-team/)
- [WCAG 2.2 criterion for 'Consistent Help'](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html)

## Why

The new WCAG 2.2 Level A criterion 'Consistent Help' says:

> If a [Web page](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html#dfn-web-page) contains any of the following help [mechanisms](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html#), and those mechanisms are repeated on multiple Web pages within a [set of Web pages](https://www.w3.org/WAI/WCAG22/Understanding/consistent-help.html#dfn-set-of-web-pages), they occur in the same order relative to other page content, unless a change is initiated by the user:
> * Human contact details;
> * Human contact mechanism;
> * Self-help option;
> * A fully automated contact mechanism.

Although the intent of the criterion is that the contact information itself is consistently ordered with respect to the rest of the page (i.e. a help link is always in the same place in the footer, as described in #2871) we can go beyond this by advising that teams provide this information in a consistent order every time it appears.

For example, if a service has three contact mechanisms—phone, email address, post—these should be listed in the same order and format on all pages where it appears, such as the service start page, support pages, contact page, etc. 

## Who needs to work on this
- Content designer or technical writer

## Who needs to review this
- Accessibility specialist
- Content designer or technical writer

## Done when

- [ ] Guidance has been updated

**Comment by dav-idc** [comment]
This issue appears to already be covered by the existing planned guidance updates in the WCAG 2.2 content updates main guidance doc.

Here's the existing section that covers 'Contact a department or service team' and consistent help: https://docs.google.com/document/d/1nIxIKgVUheC8zvVdqWIcqtVPCl7oTN4hv11R9rDwdvM/edit#heading=h.arawptfmd6cg

As a result I'm closing this issue as a duplicate of the main WCAG 2.2 guidance issue:

- https://github.com/alphagov/govuk-design-system/issues/2729
- https://github.com/alphagov/govuk-design-system/pull/3090
